tool/docker/devel/native/ubuntu/cuda10/py3/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# 
+# Change tags to build with different cuda/cudnn versions:
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+
+# install dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        git \
+        build-essential \
+        autoconf \
+        libtool \
+        libprotobuf-dev \
+        libopenblas-dev \
+        libpcre3-dev \
+        protobuf-compiler \
+        wget \
+        swig \
+        openssh-server \
+        python3-dev \
+        python3-pip \
+        python3-setuptools \
+        libgoogle-glog-dev \
+        sudo \
+    && apt-get clean \
+    && apt-get autoremove \
```

Comment:
do you need to install icc and icpc to compile mkldnn?

Reply:
hi i will review this. for now with "manual fixing cmake file" i can execute mkldnn test